src/Stepper/Stepper.js

```diff
@@ -0,0 +1,136 @@
+// @flow
+// @inheritedComponent Paper
+
+import React, { Children } from 'react';
+import type { Element, Node, ChildrenArray } from 'react';
+import classNames from 'classnames';
+import withStyles from '../styles/withStyles';
+import Paper from '../Paper';
+import StepConnector from './StepConnector';
+import Step from './Step';
+
+export const styles = (theme: Object) => ({
+  root: {
+    display: 'flex',
+    padding: theme.spacing.unit * 3,
+    // TODO: Guidelines state 24px, should we use `theme.spacing.unit * 3` or 24px?
```

Comment:
Looks good this way :). It's helping with vertical rythme.

Reply:
👍